map_view/show-location-history/README.md

```diff
@@ -6,37 +6,34 @@ Display your location history on the map.
 
 ## Use case
 
-You can track device location history and display it as lines and points on the map. The history can be used to visualize how the user moved through the world, to retrace their steps, or to create new feature geometry. An unmapped trail, for example, could be added to the map using this technique.
+You can track device location history and display it as lines and points on the map. The history can be used to visualize how the user moved through the world, to retrace their steps, or to create new feature geometry. For example, an unmapped trail could be added to the map using this technique.
 
 ## How to use the sample
 
-Click the button to start tracking your location, which will appear as points on the map. A line will connect the points for easier visualization. Click the button again to stop updating the location history.
+Click the button to start tracking your location, which will appear as points on the map. A line will connect the points for easier visualization. Click the button again to stop updating the location history. This sample uses a simulated data source to allow the sample to be useful on desktop/non-mobile devices.
```

Comment:
I think it'd be more percise to say `This sample uses a simulated data source to allow the sample to be useful on devices without a location data source`. This is since just because a device is desktop, doesn't mean it won't have the means of getting it's location. For example most desktop devices can get a rough location via WiFi, etc.

Reply:
Sounds good! I could also add this to the use case: `You can use an existing dataset in place of a connection to a GPS device. A simulated data source is useful if you're giving a demonstration or are working with previously collected data.` from [Setting up the simulation](https://desktop.arcgis.com/en/arcmap/10.3/map/display-gps-data/setting-up-the-simulation.html). But that could be redundant!